pkg/devfile/parser/data/common/command_helper.go

```diff
@@ -59,6 +59,16 @@ func (dc DevfileCommand) GetExecWorkingDir() string {
 	return ""
 }
 
+// IsExec checks if the command is an exec command
+func (dc DevfileCommand) IsExec() bool {
+	isExec := false
+	if dc.Exec != nil {
+		isExec = true
+	}
+
+	return isExec
```

Comment:
can we just `return dc.Exec!=nil`?

Reply:
Actually, `command.Composite != nil` and `command.Exec != nil` can be used. why we need an extra function?

Reply:
yes

Reply:
It looks prettier, if you're very opposed to it; I can remove it. These other funcs has been in the code, so I would have to update them all.

Reply:
No problem,  I am fine with this.

Reply:
I've removed it. I think more ppl will question it and not use it. Its not worth explaining myself everytime 😄